package.json

```diff
@@ -41,8 +41,8 @@
     "font-awesome": "4.7.0",
     "mathjax": "3.0.1",
     "mocha": "10.2.0",
-    "sass": "1.63.3",
     "puppeteer": "20.5.0",
+    "sass": "1.63.3",
```

Comment:
How has this saas dependency entry been moved within the file?
I've been careful to keep my fork of the project main in sync, and to rebase the branch on top of that.
This is the only non-doc file modified.
Apologies if I've done something wrong here.

Reply:
No worries, `npm` automatically sorts dependencies alphabetically.